workers/data_refinery_workers/processors/management/commands/run_processor_job.py

```diff
@@ -0,0 +1,48 @@
+from django.core.management.base import BaseCommand
+from data_refinery_common.job_lookup import ProcessorPipeline
+from data_refinery_common.logging import get_and_configure_logger
+from data_refinery_workers.processors.array_express import affy_to_pcl
+from data_refinery_workers.processors.transcriptome_index import build_transcriptome_index
+from data_refinery_workers.processors.no_op import no_op_processor
+
+
+logger = get_and_configure_logger(__name__)
+
+
+# Test this.
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--job-name",
+            type=str,
+            help=("The processor job's name. Must be enumerated in job_lookup."))
+        parser.add_argument(
+            "--job-id",
+            type=int,
+            help=("The processor job's ID."))
+
+    def handle(self, *args, **options):
+        if options["job_id"] is None:
+            logger.error("You must specify a job ID.")
+            return 1
```

Comment:
Why 0/1 rather than T/F?

Also, it looks like this file is copy pasted a few times maybe, could it be made generic?

Reply:
Same as above for the 0/1 things. I'm not actually entirely sure what you mean by this file being copied a few times, but except for checking that a `job_id` parameter is passed in I don't think too much functionality is repeated.